### ISSUE TITLE
some questions about paper and code

### ISSUE THREAD

**Issue by nlilin** [Zhaozixiang1228/MMIF-CDDFuse#27]
During the training phase, your paper does not appear to calculate the loss of the image gradient, but this is in the code; Although there are mse_loss and simm_loss of calculating images in the code, they do not participate in backpropagation; In addition, I did not see the lite_transformer mentioned in the paper in the code. I am very confused by this, and I appreciate it if you can correct and answer these questions.
